andela/persephone-ah-backend
# #166816102 Ft reset password

#### What does this PR do?
Add password feature to backend logic

#### Description of Task to be completed?
> - Add users/forgot_password endpoint
> - Add users/password_reset route

#### How should this be manually tested?
> - Clone this branch and cd into its root folder
> - Run npm install and npm start
> - Ensure your .env file is configured following the .env.example
> - Open your postman and hit the endpoint with an email that is existing
> - You can create a new account by making a post request to the signup route
> - Use the same email to make a post request to the user/forgot_password route
> - Then check your mailtrap.io box for a link to reset password
> - Then make a patch request using that link with your new password
> - Then try login in with the new password
> - If all went well then the reset was successful

#### Any background context you want to provide?
For purposes of development, you would need to sign up an account with mailtrap.io in order to get each mail sent to you.

#### What are the relevant pivotal tracker stories?
link to pt-story [#166816102](https://www.pivotaltracker.com/story/show/166816102)

#### Screenshots (if appropriate)

#### Questions:
